packages/hint-amp-validator/src/meta.ts

```diff
@@ -0,0 +1,20 @@
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hintscope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+
+const meta: HintMetadata = {
+    docs: {
+        category: Category.performance,
+        description: `Require HTML page to be AMP valid.`,
```

Comment:
A few months back @alrra and I were talking about how most of the hints started with `Require XXX` and how we wanted to change that because it looks weird on the site 😅

I think now is a good time. I'll create a new comment with a table so it's easier to digest and propose changes.